profiles/arch/arm/package.use.mask

```diff
@@ -2,6 +2,10 @@
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
+# Thomas Deutschmann <whissi@gentoo.org> (2016-12-04)
+# sci-libs/gdal[mdb] requires dev-java/jackcess which is not keyworded
+>=sci-libs/gdal-2.0.3 mdb
```

Comment:
For arm we have to keyword commons-lang and commons-logging first. So I will not keyword for jackcess for arm at the moment. Feel free to test & keyword all the dependencies once the package is in the repository.